src/content/examples/CustomScrollbars/style.scss

```diff
@@ -1,32 +1,39 @@
 #example {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-
-  .scrollbars {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 1fr;
-    align-items: center;
-    justify-items: center;
-    width: 400px;
-    height: 400px;
-
-    .scrollbar {
-      border: 2px dashed $color-gray;
-      border-radius: 4px;
-      width: 150px;
-      height: 150px;
-      font-size: 0.8em;
-
-      .scrollbar-inner {
-        padding: 4px 8px;
-      }
+  display: grid;
+  place-items: center;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  box-sizing: border-box;
+  padding: 20vh 30%;
+
+  @include media-max('md') {
+    padding: 5%;
+  }
+
+  .scrollbar {
+    border: 2px dashed $color-gray;
+    border-radius: 4px;
+    width: 150px;
+    height: 150px;
+    font-size: 0.8em;
+
+    .scrollbar-inner {
+      padding: 4px 8px;
+    }
+
+    table {
```

Comment:
I think that this is a leftover from your tests maybe?

Reply:
Yes :D good catch! I've created a crazy stress-test full of DOM nodes directly in the example page and then removed it